jenkins/README.md

```diff
@@ -0,0 +1,105 @@
+## Configure Jenkins slave to master
+
+### Install Java-11 based on OS
+rhel8
+```commandline
+sudo yum install java-11-openjdk-devel
+```
+amazon-linux
+```commandline
+yum install java-11-amazon-corretto-devel.x86_64
+```
+
+### Install Docker on CentOS/ Fedora
+Centos: https://docs.docker.com/engine/install/centos/
+Fedora: https://docs.docker.com/engine/install/fedora/
+
+### Installing on Fedora
+```
+sudo dnf -y install dnf-plugins-core
+sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+sudo dnf install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+sudo systemctl start docker
+```
+
+### Add Jenkins user
+```commandline
+useradd jenkins -U -s /bin/bash
+```
+Set password
+```commandline
+passwd jenkins 
+```
+ex: test123
+
+### Add Jenkins to the sudoers file
+
+$ vi /etc/sudoers
+```commandline
+jenkins     ALL=(ALL)       NOPASSWD: ALL
+```
+
+### Create a Key-Pair
+```commandline
+ssh-keygen
+cat .ssh/id_rsa.pub >> .ssh/authorized_keys
+```
+
+### Connect to Jenkins Master with your node
+
+Open Jenkins master
+1. Manage Jenkins
+![Manage Jenkins](../images/jenkins/manage_jenkins.png)
+2. Manage Nodes and Clouds
+![ManageNodes](../images/jenkins/manage_nodes.png)
+3. New Node
+![NewNode](../images/jenkins/new_node.png)
+4. Add details
+![Details](../images/jenkins/slave_node.png)
+5. Configure Node
+![Config](../images/jenkins/jenkins_config_file.png)
+
+Click on add to add jenkins user and private_key
+![PrivateKey](../images/jenkins/add_creds.png)
+
+
+Give permissions to **jenkins** user to run docker daemon
+```commandline
+sudo chown jenkins:jenkins /var/run/docker.sock
+```
+Check docker working from jenkins user
+```commandline
+sudo su - jenkins
+docker images
+```
+
+Now you are ready to run the cloud-governance policies
+
+### Run the ElasticSearch, Grafana, Kibana as a container within the same network
+
+#### Using the docker engine
+```commandline
+# detached mode
+docker-compose up -d
+```
+
+#### Using the podman 
+
+create elasticsearch & grafana local persistence Directories
+
+Create and Allow Permissions
+```commandline
+mkdir -p /tmp/grafana
+mkdir -p /tmp/elasticsearch
```

Comment:
Update it as following:

CLOUD_GOVERNANCE_PATH=""

mkdir -p "$CLOUD_GOVERNANCE_PATH"/elastic/
mkdir -p "$CLOUD_GOVERNANCE_PATH"/grafana/

# chmod solved ci_pod degraded
chmod 775 -R "$CLOUD_GOVERNANCE_PATH"/elastic/
chmod 775 -R "$CLOUD_GOVERNANCE_PATH"/grafana/

For more details [see](https://github.com/redhat-performance/benchmark-runner/blob/ecdb4396dd1b1f708ff1a5d794f043aa8d449c63/ci_pod/create_ci_pod.sh)